gulpfile.js

```diff
@@ -1,5 +1,6 @@
 const { src, watch, series, dest } = require("gulp");
-const sass = require("gulp-sass");
+// const sass = require("gulp-sass");
+const sass = require("gulp-sass")(require("node-sass"));
```

Comment:
Directly from the docs: https://www.npmjs.com/package/gulp-sass#user-content-migrating-to-version-5

Reply:
Shouldn't we also add `node-sass` to package.json?

Reply:
@lloan Good catch – I revised this require statement to use just `sass`!